bugbug/bug_snapshot.py

```diff
@@ -868,17 +869,22 @@ def parse_flag_change(change):
     return bug
 
 
-def get_inconsistencies(bugs):
-    inconsistencies = []
+def _try_rollback(bug):
+    try:
+        rollback(bug, do_assert=True)
+        return None
+    except Exception:
+        logger.exception("Failed to rollback bug %s", bug["id"])
+        return bug
```

Comment:
Returning `None` does the work of distinguishing between successful and failed rollback. However, it is not intuitive why the function returns `None`  when successful and returns a bug when it fails. Thus, I suggest to be more consistent with what the `rollback()` would do:

```suggestion
def _try_rollback(bug):
    try:
        return rollback(bug, do_assert=True), True
    except Exception:
        logger.exception("Failed to rollback bug %s", bug["id"])
        return bug, False
```

Reply:
Good idea, done!